ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/frame/GameFrame.java

```diff
@@ -81,7 +81,12 @@ void onWindowCloseKeyboardRequest() {
 
     public void addGame(@NotNull Game game) {
         GamePanel panel = new GamePanel(getGlassPane(), game);
-        mTabbedPane.addTab(game.getGameType() + mTabbedPane.getTabCount(), panel);
+        if(mTabbedPane.getTabCount() != 0) {
```

Comment:
We don't need this change, and even if we decide we want it, it needs to be in a separate commit